packages/arb-token-bridge-ui/src/hooks/useTheme.ts

```diff
@@ -1,16 +1,30 @@
 import { useEffect } from 'react'
+import useLocalStorage from '@rehooks/local-storage'
 
-import { useArbQueryParams } from './useArbQueryParams'
+export const classicThemeKey = 'arbitrum-classic-theme'
 
-export const useTheme = () => {
-  const [{ theme }, setQueryParams] = useArbQueryParams()
-
-  const setTheme = (_theme: string) => {
-    setQueryParams({ theme: _theme })
+export const THEME_CONFIG = [
+  {
+    id: 'space',
+    label: 'Space',
+    queryParam: '',
```

Comment:
do we still need this property now that we're not using query params anymore?

Reply:
Yeah, we don't. `ID` is sufficient. Removed and handled accordingly.